### PR TITLE
test: increase timeouts for tests with operate/zeebe roundtrip

### PR DESCRIPTION
**Pull request by pedesen**
## Description

e2e tests are often failing in CI due to timeouts. Some of these failing tests were identified as roundtrip tests.

Roundtrip tests are tests which follow these steps:
1. an action is triggered in the Operate UI
2. frontend calls backend API
3. backend sends a command to Zeebe
4. backend imports updated data
5. frontend polls for updated data
6. frontend renders updated data in Operate UI

### Changes in this PR
- tag roundtrip tests with `@roundtrip`
- annotate roundtrip tests with `test.slow` to increase the test timeout from 30 to 90 seconds.
- stabilized batchMoveModification e2e test by reloading page until data is updated.

### CODE REVIEW

File: operate/client/e2e-playwright/tests/processInstance.spec.ts [hunk 1]
```diff
@@ -92,8 +88,11 @@ test.beforeAll(async ({request}) => {
 });
 
 test.describe('Process Instance', () => {
-  test('Resolve an incident', async ({page, processInstancePage}) => {
-    test.setTimeout(DEFAULT_TEST_TIMEOUT + 3 * 15000); // 15 seconds for each applied operation in this test
```
Comment (pedesen) [inline operate/client/e2e-playwright/tests/processInstance.spec.ts:96]: I removed this in favor of `test.slow()`.

Before: 75s timeout (30s + 45s)
After: 90s timeout (30s * 3)

File: operate/client/e2e-playwright/tests/processInstance.spec.ts [hunk 2]
```diff
@@ -199,8 +198,8 @@ test.describe('Process Instance', () => {
     ).toBeVisible();
   });
 
-  test('Cancel an instance', async ({page, processInstancePage}) => {
-    test.setTimeout(DEFAULT_TEST_TIMEOUT + 1 * 15000); // 15 seconds for each applied operation in this test
```
Comment (pedesen) [inline operate/client/e2e-playwright/tests/processInstance.spec.ts:203]: I removed this in favor of `test.slow()`.

Before: 45s timeout (30s + 15s)
After: 90s timeout (30s * 3)

File: operate/client/e2e-playwright/tests/processInstanceVariables.spec.ts [hunk 3]
```diff
@@ -47,8 +47,8 @@ test.beforeAll(async ({request}) => {
 });
 
 test.describe('Process Instance Variables', () => {
-  test('Edit variables', async ({page, processInstancePage}) => {
-    test.setTimeout(DEFAULT_TEST_TIMEOUT + 1 * 15000); // 15 seconds for each applied operation in this test
```
Comment (pedesen) [inline operate/client/e2e-playwright/tests/processInstanceVariables.spec.ts:51]: I removed this in favor of `test.slow()`.

Before: 45s timeout (30s + 15s)
After: 90s timeout (30s * 3)

File: operate/client/e2e-playwright/tests/processInstanceVariables.spec.ts [hunk 4]
```diff
@@ -81,8 +81,12 @@ test.describe('Process Instance Variables', () => {
     await expect(page.getByText('"editedTestValue"')).toBeVisible();
   });
 
-  test('Add variables', async ({page, processInstancePage, processesPage}) => {
-    test.setTimeout(DEFAULT_TEST_TIMEOUT + 1 * 15000); // 15 seconds for each applied operation in this test
```
Comment (pedesen) [inline operate/client/e2e-playwright/tests/processInstanceVariables.spec.ts:85]: I removed this in favor of `test.slow()`.

Before: 45s timeout (30s + 15s)
After: 90s timeout (30s * 3)

**Review by vitorwtavares — APPROVED** [review]

**Pull request merged**